cpp/thirdparty/CMakeLists.txt

```diff
@@ -66,40 +66,21 @@ endif()
 
 # ## BOOST
 if(MEMILIO_USE_BUNDLED_BOOST)
-    message(STATUS "Downloading Boost library. This may take several minutes. This is the time to get yourself a coffee. :-)")
+    message(STATUS "Downloading Boost library")
+
+    string(REPLACE "." "_" MEMILIO_BOOST_VERSION_UNDERSC "${MEMILIO_BOOST_VERSION}")
 
     include(FetchContent)
     FetchContent_Declare(boost
-    GIT_REPOSITORY https://github.com/boostorg/boost.git
-    GIT_TAG boost-${MEMILIO_BOOST_VERSION})
+        #don't use the URL from github, that download isn't complete and requires more setup (subrepositories, bootstrapping)
+        URL https://archives.boost.io/release/${MEMILIO_BOOST_VERSION}/source/boost_${MEMILIO_BOOST_VERSION_UNDERSC}.tar.gz

```

Comment:
Can a "normal" Windows install deal with .tar.gz archives? Or is that supported through some other required package like VS?

Reply:
cmake has a bunch of unix command line tools built in, `tar` is one of them, see [here](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E-arg-tar). I think this would have shown up before if it was a problem.